test/fiddle/test_pointer.rb

```diff
@@ -20,16 +20,28 @@ def test_malloc_free_func_int
       assert_equal free.to_i, Fiddle::RUBY_FREE.to_i
 
       ptr  = Pointer.malloc(10, free.to_i)
-      assert_equal 10, ptr.size
-      assert_equal free.to_i, ptr.free.to_i
+      refute ptr.freed?
```

Comment:
Do we need test `freed?` here?
It seems that you add separated tests for it.

Reply:
Tidied up.